Add basic i18n for list and single

doliG opened the pull request:
Add i18n support in templates. close #29 

Note: we should also update the repo wiki (but I don't have the right to).

LukasJoswiak commented on README.md:
```diff
@@ -12,6 +12,7 @@ Etch is a simple, responsive theme for [Hugo](https://gohugo.io) with a focus on
 * Syntax highlighting with customizable theme.
 * Dark theme which automatically adjusts based on users' setting ([example](https://github.com/LukasJoswiak/etch/wiki/Dark-mode)).
 * No external dependencies, no JavaScript, no web fonts.
+* Internationalization friendly: use default english translations or create your own
```
```suggestion
* Internationalization friendly: use default English translations or create your own.
```

doliG replied:
Done

LukasJoswiak commented on i18n/en.yaml:
```diff
@@ -0,0 +1,15 @@
+# Learn how to use Date format (date, created, updated)
+# -> https://gohugo.io/functions/dateformat/
+
+posts:
+  title:
+    other: "Posts"
+  date:
+    other: "Jan, 2 2006"
+
+
+post:
+  created:
+    other: "Jan, 2 2006"
+  updated:
+    other: "Updated Jan, 2 2006"
```
Can you convert this to use TOML instead of YAML? And move the comma after the day of the month instead of the month (ie `Jan 2, 2006`).

doliG replied:
Done

LukasJoswiak commented on layouts/_default/li.html:
```diff
@@ -1,6 +1,6 @@
-        {{ .Title }}
-        <small><time>{{ .Date.Format "Jan 2, 2006" }}</time></small>
-    </a>
+  <a href="{{ .Permalink }}">
+    {{ .Title }}
+    <small><time>{{ .Date | time.Format (i18n "posts.date") }}</time></small>
+  </a>
```
Use four spaces here instead of two, to remain consistent with the rest of the codebase.

LukasJoswiak reviewed (changes requested):
Thanks, this will be very useful! Just a few small nits :)

LukasJoswiak commented:
I added a (draft) wiki page at https://github.com/LukasJoswiak/etch/wiki/i18n. Let me know if you have feedback.

doliG commented:
I've added the changes you requested. Tell me if it's good for you :)

> I added a (draft) wiki page at https://github.com/LukasJoswiak/etch/wiki/i18n. Let me know if you have feedback.

It's perfect, simple and straight to the point 👌

LukasJoswiak reviewed (approved):
Looks great!
